### PR TITLE
Fix busy event emission on fixed pool:

### LINKED ISSUE #245: BUG_Busy event emission count
**Describe the bug**
The busy event emission count is off by one on fixed pool. 

**Steps to reproduce the behavior**
Run 'Verify that new workers are created when required, max size is not exceeded and that after a while new workers will die' with `expect(poolBusy).toEqual(max)`

**Info**

| Tool      | Version       |
| --------- | ------------- |
| Poolifier | v2.x.x        |
| Node      | v14.x.x       |
| OS        | mac           |

### PR DESCRIPTION
**Pull request by jerome-benoit**
Always trigger the event after the task submission.

Close #245.

Signed-off-by: Jérôme Benoit <jerome.benoit@sap.com>

## PR Checklist

- [x] Please add a description of your changes.
- [x] We need your changes to come with unit tests in order to keep this project in quality and easy to maintain.
- [ ] If your changes contain any new feature please be sure to document it.
- [ ] Please add a link to the open issue or task that this pull request will resolve.
- [x] Add a note to the changelog to indicate the change.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

If yes please indicate it and migration info into the CHANGELOG.md file.

---

### CODE REVIEW

File: tests/pools/abstract/abstract-pool.test.js
Comment (Shinigami92) [inline tests/pools/abstract/abstract-pool.test.js:140]: **question:** @jerome-benoit Could you write (at least) a short explanation **why** we need to add here 1 on top of `numberOfWorkers`?
I currently don't understand it just by reading the code within the review :slightly_smiling_face:

Comment (jerome-benoit) [reply tests/pools/abstract/abstract-pool.test.js:142]: Done.

**Review by Shinigami92 — APPROVED** [review]

**Comment by Shinigami92** [comment]
IMO you should wait on a second approval from @pioardi

**Comment by jerome-benoit** [comment]
> IMO you should wait on a second approval from @pioardi

Sure, but if he does not respond in a timely manner (say one week), I will merge it without his approval.

**Review by pioardi — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**
